### PR TITLE
Feature/129425 search filter tab js

### PR DESCRIPTION
**Pull request by paulmskim**
https://central.tri.be/issues/129305

On hold until https://github.com/moderntribe/the-events-calendar/pull/2588 and https://github.com/moderntribe/the-events-calendar/pull/2595 are merged in.

### CODE REVIEW

File: src/views/v2/top-bar/actions.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+ * See more documentation about our views templating system.
+ *
+ * @link {INSERT_ARTCILE_LINK_HERE}
+ *
+ * @version TBD
+ *
+ */
+?>
+<div class="tribe-events-c-top-bar__actions">
+	<?php $this->template( 'top-bar/actions/hide-recurring' ); ?>
```
Comment (juanfra) [inline src/views/v2/top-bar/actions.php]: The idea is that this piece was moved to PRO as recurring events is only a PRO functionality.

Comment (paulmskim) [reply src/views/v2/top-bar/actions.php]: The hide recurring file in TEC is blank. It hooks in from PRO.

Comment (juanfra) [reply src/views/v2/top-bar/actions.php]: Yep, that's what I mean, shouldn't we remove completely the hidden file and hook the PRO template at another point? I mean, it's the only action of the `top-bar__actions`.

Comment (bordoni) [reply src/views/v2/top-bar/actions.php]: We want to make sure we have an empty content file for the actions bar as a whole, not specific to hide recurring.

Comment (paulmskim) [reply src/views/v2/top-bar/actions.php]: done, changed back to actions.

**Comment by paulmskim** [comment]
This PR is still on hold until https://github.com/moderntribe/the-events-calendar/pull/2588 is merged in.

**Pull request merged**